themes/perfect-dusk.json

```
@@ -296,6 +296,7 @@
         "terminal.ansi.bright_white": "#ffffff",
         "terminal.ansi.bright_yellow": "#ebff00",
         "terminal.ansi.cyan": "#00fff7",
+        "terminal.ansi.bright_black": "#525561ff",
```

Thank you very much for that fix! I think the gray we already use for the `"border"` would be perfect here. That way, it'll blend in nicely with the rest of the color scheme 😄 

```suggestion
        "terminal.ansi.bright_black": "#444",
```